torchvision/utils.py

```diff
@@ -1,10 +1,13 @@
+from typing import Union, Optional, Sequence, Tuple
+import io
 import torch
 import math
 irange = range
 
 
-def make_grid(tensor, nrow=8, padding=2,
-              normalize=False, range=None, scale_each=False, pad_value=0):
+def make_grid(tensor: Union[torch.Tensor, Sequence[torch.Tensor]], nrow: int = 8,
+              padding: int = 2, normalize=False, range: Optional[Tuple[int, int]] = None,
```

Comment:
nit:  `normalize: bool = False`, although I would assume that mypy can infer the type annotation from the default argument